src/components/People/People.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import { useMatch } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable';
+
+export const People: React.FC = () => {
+  const match = useMatch('people/:slug');
+  const selectedPersonSlug = match?.params.slug;
+
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isPeopleLoading, setIsPeopleLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isNoPeople, setIsNoPeople] = useState(false);
```

Comment:
It's a redundant variable. Just check the people array length

Reply:
Take attention to this comment and try to fix it.